src/pkcs11/versioninfo-pkcs11.rc.in

```diff
@@ -37,7 +37,7 @@ BEGIN
 END
 
 #ifndef __MINGW32__
-2 RT_MANIFEST "@abs_top_srcdir@\\src\\pkcs11\\opensc-pkcs11.dll.manifest"
+2 RT_MANIFEST "..\\..\\src\\pkcs11\\opensc-pkcs11.dll.manifest"
```

Comment:
I changed this to @abs_top_srcdir@ intentionally to support out-of-source builds (Meson does not support in-source builds by design). Why have you changed it back?

Reply:
I tried to hint this in the commit message. All previous CI runs failed, for example [this one](https://ci.appveyor.com/project/frankmorgner/opensc/builds/49975421/job/ff9mfopuyq2b0i2b), because when we use nmake, we run `./configure` in a cygwin environment, which replaces `abs_top_srcdir` with the cygpath. However, when we build the files with nmake, only actual windows paths must be used.

Reply:
OK, I see. We have three options here:
1. Use native Cygwin's `windres` to compile resources;
2. Introduce a new substitution variable with a correct path (is it possible to do in `configure.ac`?). On Cygwin we should call `cygpath` to get the correct path;
3. Close this PR and create separate .rc files with manifests for Meson only.

I personally vote for the third point, because it is relatively simple to implement (for me). What do you think?